deploy/helm/magda/charts/indexer/templates/deployment.yaml

```diff
@@ -12,6 +12,7 @@ spec:
       labels:
         service: indexer
     spec:
+      priorityClassName: magda-8
```

Comment:
The indexer should probably sit a level below `api`s given that it's fine for it to be bumped at any time (it'll resume its hook and catch up when it restarts)